docs/sphinx/using/hardware.rst

```diff
@@ -0,0 +1,202 @@
+
+Specifying the QPU
+'''''''''''''''''''
+
+By default, the IonQ target will use the :code:`simulator` QPU.
+To specify which IonQ QPU to use, set the :code:`qpu` parameter.
+A list of available QPUs can be found `in the API documentation
+<https://docs.ionq.com/#tag/jobs>`_.
+
+.. code:: python
+
+    cudaq.set_target("ionq", qpu="qpu.aria-1")
+
+Example
+........
+
+.. literalinclude:: ../examples/python/ionq
+   :language: python
+
+
+Quantinuum
+==================================
+
+Setting Credentials
+```````````````````
+
+Programmers of CUDA Quantum may access the Quantinuum API from either
+C++ or Python. Quantinuum requires a credential configuration file
+in your ``$HOME`` directory. This may be generated as follows, replacing
+the ``email`` and ``credentials`` in the first line with your Quantinuum
+account details.
+
+.. code:: bash
+
+    # You may need to run: `apt-get update && apt-get install curl jq`
+    curl -X POST -H "Content Type: application/json" -d '{ "email":"<your_alias>@email.com","password":"<your_password>" }' https://qapi.quantinuum.com/v1/login > $HOME/credentials.json
+    id_token=`cat $HOME/credentials.json | jq -r '."id-token"'`
+    refresh_token=`cat $HOME/credentials.json | jq -r '."refresh-token"'`
+    echo "key: $id_token" >> $HOME/.quantinuum_config
+    echo "refresh: $refresh_token" >> $HOME/.quantinuum_config
+    export CUDAQ_QUANTINUUM_CREDENTIALS=$HOME/.quantinuum_config
+
+
+C++
+````
+
+For C++, the ``--target`` argument may be set to "quantinuum". ``nvq++`` will grab
+the credentials from your home directory, authenticate them with the Quantinuum API,
+and submit any quantum kernel executions to the hardware. By default, the QPU is set
+to Quantinuum's syntax checker. This is helpful for determining the validity of your
+kernel before submitting to a physical QPU. 
+
+.. code:: bash
+
+    nvq++ --target quantinuum src.cpp ...
+
+
+Specifying the QPU
+'''''''''''''''''''
+
+The ``quantinuum`` target will select a Quantinuum emulator by default.
+To specify a different QPU, this may be done in ``nvq++`` as
+
+.. code:: bash
+
+    nvq++ --target quantinuum --quantinuum-machine H1-2 src.cpp ...
+
+where ``H1-2`` is an example of a physical QPU. Hardware specific
+emulators may be accessed by appending an "E" to the end (e.g, ``H1-2E``).
+For a comprehensive list of available backends, refer to your `Quantinuum user account
+<https://um.qapi.quantinuum.com/user>`_.
+
+
+Example
+.......
+
+.. literalinclude:: ../examples/cpp/quantinuum
+    :language: cpp
+
+
+Python 
+```````
+
+In python, the target may be controlled with the ``cudaq.set_target()``
+function. This will set the target for any kernel executions within the file,
+and will go through the same credential scheme as discussed in the C++ case.
+
+.. code:: python
+
+    cudaq.set_target('quantinuum')
+
+To emulate the Quantinuum machine locally, without submitting through the cloud,
+you can also set the ``emulate`` flag to ``True``.
```

Comment:
We might want to clarify that this is targeting the application to the quantinuum backend such that the compiler emits any target specific diagnostics, but it isn't emulating noise.